### ISSUE TITLE
Thanks for updating the link but still there is some issue while extracting the zip file (its empty)

### ISSUE THREAD

**Issue by sahilshukla3003** [EliaFantini/2D-Priors-for-3D-human-reconstruction#3]
![Screenshot from 2024-06-11 18-09-57](https://github.com/EliaFantini/2D-Priors-for-3D-human-reconstruction/assets/124785012/5eb02909-1fc3-4a31-bc47-9820d2363f30)

**Comment by sahilshukla3003** [comment]
Kind request , can you provide it ASAP..

**Comment by Aditya-U-Pawar** [comment]
HI zip file is currupted or empty please provide the correct link

**Comment by EliaFantini** [comment]
Thanks for your patience. This project was a university project and all the files was stored in the university server to which I have no more access as I am not a student anymore, therefore the original link expired. We still have some files remaining on our computer but unfortunately, the original RAR file was corrupted. I have retrieved the "eval_masking" and "tta_clip" folders, but "eval" folder is missing. I have updated the link with a file containing the folders I found and some more files that you hopefully will find useful, sorry for the inconvenience

**Comment by sahilshukla3003** [comment]
so what does the eval contains like what is the process to create the eval folder

**Comment by EliaFantini** [comment]
I don't remember precisely as the project is from one year ago. I believe it could be eval_baseline, which I uploaded. Anyway if you want to be sure, the structure would be very similar to the folders I uploaded and can be reconstructed by looking at what the code asks. test.sh script launches eval.py, which has no reference to this folder. Such folder is accessed in  clip_loss_tta.py, as DATA_PATH, and current default there is '/scratch/izar/fantini/results/tta_clip' , so if needed eval will likely have similar structure. Anyway check carefully the code, sorry for the inconvenience.

**Comment by EliaFantini** [comment]
The purpose is of course to run evaluation, so you run the model, you get the output, you compare it to groundtruth and calculate some metrics to evaluate performance, so what was in that folder was likely input models and masks to input in the model and some groundtruth to compare results. You can redo this with the provided dataset data, there's code to create masks and everything

**Comment by sahilshukla3003** [comment]
Thanks for your reply ! I will follow the step as mention in the repo and will try to generate the 3d reconstruction. Just a kind request can you please add a drive link which contains the outputs that are genereted by using inference for unseen data so that we can get an idea regarding its overall performance.

**Comment by EliaFantini** [comment]
https://drive.google.com/file/d/1RwbEKfXPHQ_uDVzBvSOX_U_BQCkv026C/view?usp=sharing

Here it is. As said, it's a university project from 1 year ago, so many files we lost/deleted them, especially output which are unnecessary and take a lot of space. But I found a few output results and I uploaded them. Please check also the [report](https://github.com/EliaFantini/2D-Priors-for-3D-human-reconstruction/blob/main/report.pdf) , as you can see there, our final results were not great, generalization was not really improved a lot, and this is related to 2D errors in images. Regarding its robustness to data outside the training data, it will be the same as the standard PIFu as the scope of our research was not that. What are you trying to achieve/research?

**Comment by sahilshukla3003** [comment]
Drive link seems to be corrupted again and I am try to achieve better 3d reconstruction than pifu/pifuHD with the available resources.

**Comment by EliaFantini** [comment]
Sorry which link/drive is corrupted? I checked and they work. Anyway to be honest I don't think our work can really help you with the objective you have. Our objective was to investigate ways to improve robustness to image corruptions, very different, therefore our material doesn't really apply. I suggest you to refer to original PiFu repositories and work on that
